### DISCUSSION TITLE
How to add an external link as a left menu item

### DISCUSSION THREAD

**Discussion by KyleMaas** [mediacms-io/mediacms#665] [Q&A]
How could I add an item to the left menu which would be a link to an external site?

**Comment by KyleMaas** [comment]
I should clarify: I'd ideally like to place it up near the top of the menu, just below Home, and would like to link it to another subdomain.

**Comment by mgogoulos** [comment]
Have you found the place? There is an approach you can follow on https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#13-how-to-add-a-static-page-to-the-sidebar but I believe there must be other (better) ways right now without lots of changes/refactoring...

**Comment by KyleMaas** [comment]
Yes, but I did it a bit differently.  Working on it in the Owncast feature.

**Comment by SaturnTeamDev** [comment]
I just added an updated method in discussion #1049.
